pennylane/devices/default_tensor.py

```diff
@@ -151,28 +181,38 @@ class DefaultTensor(Device):
     This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, other devices like ``default.qubit`` may be more suitable.
 
     The backend uses the ``quimb`` library to perform the tensor network operations, and different methods can be used to simulate the quantum circuit.
-    Currently, only the Matrix Product State (MPS) method is supported, based on ``quimb``'s ``CircuitMPS`` class.
+    Currently, the supported methods are Matrix Product State (MPS), based on the ``quimb``'s ``CircuitMPS`` class, and Exact Tensor Network (TN), based on the ``quimb``'s ``Circuit`` class.
```

Comment:
Minor point, but I'm not sure if the `CircuitMPS` vs `Circuit` distinction is that useful to users

Reply:
> Exact Tensor Network

Do we need the "Exact" distinction? MPS can also be exact if the hyperparameters and circuit are compatible.

Reply:
> I'm not sure if the `CircuitMPS` vs `Circuit` distinction is that useful to users

Yes, it is quite an 'internal' information, probably it is only relevant for users that are very familiar with  ``quimb``.

> Do we need the "Exact" distinction? MPS can also be exact if the hyperparameters and circuit are compatible.

I agree, we don't need such a distinction. I'll get rid of it